src/Management/src/EndpointCore/Steeltoe.Management.EndpointCore.csproj

```diff
@@ -35,4 +35,8 @@
     <PackageReference Include="Steeltoe.Common.Http" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
     <PackageReference Include="Steeltoe.Management.EndpointBase" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Should be adjusted to be either package or project reference

Reply:
@hananiel is this reference even required? I don't see it actually being used in this project